### DISCUSSION TITLE
Dynamic Power Limiter Einstellungen, PV+Batterie

### DISCUSSION THREAD

**Discussion by ar12321** [tbnobody/OpenDTU#2321] [General]
Ich betreibe am HM600 ein PV-Paneel und am zweiten Eingang eine Batterie (über Victron Laderegler).
Meine Frage, wie erkennt openDTU bzw. der Dynamic Power Limiter ob an 1 oder 2 HM600-Eingängen eine Leistung bereitstellen?
Wäre es möglich über eine Variable die Regelung zu beeinflussen (z.B. Grenzwert für PV-Leistung am 1. Eingang < 10W).
Weiterhin wäre es praktisch, wenn ohne Batterie-Lieferung (Spannung am Eingang 2 < 10V) der Power-Limiter den HM600 auf maximaler Leistung belässt.
Vielleicht gibt es schon diese Einstellungen, ich finde sie nur nicht.

**Comment by stefan123t** [comment]
@ar12321 leider kann der HM600 nur ein gemeinsames Limit für beide DC-"Kanäle"/Eingänge des HM600 vorgeben. Bei größeren Invertern mit 4-in-1 sogar nur eines für vier Eingänge. Einige nutzen deshalb einen ein-kanaligen HM300/400 für OpenDTU-OnBattery.
Ansonsten wird es schnell unschön für die OpenDTU zu rechnen und für den WR zu regeln, wenn die Eingänge keine Gleichlast, sondern eine Schieflast / -quelle aufweisen.
